src/Uno.UI.TestComparer/Comparer/TestFilesComparer.cs

```diff
@@ -147,12 +148,15 @@ select testFileIncrements
 								var currentImage = DecodeImage(folderInfo.Path);
 								var previousImage = DecodeImage(previousFolderInfo.Path);
 
-								var diff = DiffImages(currentImage.pixels, previousImage.pixels, currentImage.frame.Format.BitsPerPixel / 8);
+								if (currentImage.pixels.Length == previousImage.pixels.Length)
```

Comment:
If the image are not of the same size, I would consider them as failing, no ?

Reply:
They are, that check only happens if the images are different. We're just not generating an image diff.